### ISSUE TITLE
How to increase the batch size

### ISSUE THREAD

**Issue by hzlhzlhzlhzl** [Jamson-Zhong/Graph2Edits#8]
train_data = train_dataset.loader(
        batch_size=1, num_workers=args['num_workers'], shuffle=True)，I want to modify the batch size, but I encountered an error. Additionally, I've noticed that the memory usage is quite low when using the new version of torch. Do you have any idea why this is happening? Thank you for your assistance

**Comment by Sameer912** [comment]
If you want to increase the batch size, do it in the prepare_data.py script. 

python prepare_data.py --mode train --batch_size 64

You can pre-batchify the tensors in prepare_data.py, and during training you use batch size of 1 because tensors are already saved in batches. Hope this helps you.
